File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldChipSetDeviceSize.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnDeviceSize;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldChipSetDeviceSize extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public EnumDefnDeviceSize[] defaultValue;

  @Nullable
  public EnumDefnDeviceSize defaultVar;

  @Nullable
  public EnumDefnDeviceSize[] filterKindSet;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldChipSetDeviceSize.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnDeviceSize
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldChipSetDeviceSize : DefnFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: List<EnumDefnDeviceSize>?
  val defaultVar: EnumDefnDeviceSize?
  val filterKindSet: List<EnumDefnDeviceSize>?
}